dev-libs/libpurple/libpurple-2.13.0.recipe

```diff
@@ -0,0 +1,166 @@
+	lib:libidn$secondaryArchSuffix
+	lib:libnss3$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libnspr4$secondaryArchSuffix
+	lib:libfreebl3$secondaryArchSuffix
+	lib:libnss3$secondaryArchSuffix
+	lib:libnssckbi$secondaryArchSuffix
+	lib:libnssdbm3$secondaryArchSuffix
+	lib:libnssutil3$secondaryArchSuffix
+	lib:libsmime3$secondaryArchSuffix
+	lib:libsoftokn3$secondaryArchSuffix
+	lib:libssl3$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libffi$secondaryArchSuffix
+	lib:libpanelw$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	lib:libperl$secondaryArchSuffix
+	lib:libpython2.7$secondaryArchSuffix
+	"
+PROVIDES_devel="
+	libpurple${secondaryArchSuffix}_devel = $portVersion
+	devel:libpurple$secondaryArchSuffix = $libpurpleVersionCompat
+	devel:libgnt$secondaryArchSuffix = $libgntVersionCompat
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	libpurple$secondaryArchSuffix == $portVersion base
+	"
+PROVIDES_finch="
+	libpurple${secondaryArchSuffix}_finch = $portVersion
+	cmd:finch$secondaryArchSuffix
+	"
+REQUIRES_finch="
+	haiku${secondaryArchSuffix}
+	libpurple$secondaryArchSuffix == $portVersion base
+	lib:libffi$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgmodule_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgthread_2.0$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libidn$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libpython2.7$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libgmodule_2.0$secondaryArchSuffix
+	devel:libgthread_2.0$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libidn$secondaryArchSuffix
+	devel:libnss3$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libnspr4$secondaryArchSuffix
+	devel:libfreebl3$secondaryArchSuffix
+	devel:libnss3$secondaryArchSuffix
+	devel:libnssckbi$secondaryArchSuffix
+	devel:libnssdbm3$secondaryArchSuffix
+	devel:libnssutil3$secondaryArchSuffix
+	devel:libsmime3$secondaryArchSuffix
+	devel:libsoftokn3$secondaryArchSuffix
+	devel:libssl3$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:gettext
+	cmd:intltool_update
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python2
+	cmd:sed
+	"
+
+PATCH()
+{
+	sed -i 's/resolv/network/g' configure.ac
+}
+
+defineDebugInfoPackage libpurple$secondaryArchSuffix \
+	"$libDir"/libpurple.so.$libpurpleVersion \
+	"$libDir"/libgnt.so.$libgntVersion
+
+BUILD()
+{
+	autoreconf -fi
+	CFLAGS="-D__SVR4 -D__sun -D_XPG5 -D_BSD_SOURCE" \
+	LDFLAGS="-lbsd" \
```

Comment:
I wouldn't recommend defining macros that belongs to other operating systems. They might have behaviors that are not compatible with us. Instead, please patch the code with `#ifdef __HAIKU__` inserted where necessary.

```diff
-	CFLAGS="-D__SVR4 -D__sun -D_XPG5 -D_BSD_SOURCE" \
-	LDFLAGS="-lbsd" \
+	CPPFLAGS="-D_BSD_SOURCE" \
+	LIBS="-lbsd" \